src/screens/SendConfirmationSheet.tsx

```diff
@@ -306,12 +468,27 @@ export default function SendConfirmationSheet() {
     if (!canSubmit) return;
```

Comment:
related to my comment of gas validations https://github.com/rainbow-me/rainbow/pull/3389#pullrequestreview-1004560113 seems like we don't have any

this makes sense for normal NFTs transfers but now in this screen we are giving the option to users to send additional txs so we should validate if the user has the funds to do so

in SendScreen we use the useGas hook to get `isSufficientGas` https://github.com/rainbow-me/rainbow/blob/develop/src/screens/SendSheet.js#L108 i think it makes sense to do the same here, and block the button if ! isSufficientGas and gas is valid, since we modify the rap gas limit calling `updateTxFee` from this component as well, WDYT?

Reply:
sounds good!